// grunt-cli/Gruntfile.js
/*
 * grunt-cli
 * http://gruntjs.com/
 *
 * Copyright (c) 2012 Tyler Kellen, contributors
 * Licensed under the MIT license.
 * https://github.com/gruntjs/grunt-init/blob/master/LICENSE-MIT
 */

'use strict';

module.exports = function(grunt) {

	// Project configuration.
	grunt.initConfig({
		watch: {
			html: {
				files: ['ERS/*.html']
			},
			options: {
				livereload: true
			}
		}
		
		jshint: {
			all: [
				'Gruntfile.js',
				'lib/**/*.js',
				'bin/*',
			],
			options: {
				curly: true,
				eqeqeq: true,
				immed: true,
				latedef: true,
				newcap: true,
				noarg: true,
				sub: true,
				undef: true,
				unused: true,
				boss: true,
				eqnull: true,
				node: true,
				es5: true
			}
		},
	
		log: {
			return grunt.log.write('Grunt is installed, now do something with it!')
		}
	});

	// These plugins provide necessary tasks.
	grunt.loadNpmTasks('grunt-contrib-jshint');

	// "npm test" runs these tasks
	grunt.registerTask('test', [
		'jshint'
	]);

	// Default task.
	grunt.registerTask('default', [
		'log'
	]);

};
